### PR TITLE
Add a warning about root cert rotation in its current state

### PR DESCRIPTION
**Pull request by steeling**
Currently root cert rotation causes downtime between a users' services. We should warn them about this.

### CODE REVIEW

**Comment by zr-msft** [comment]
@openservicemesh/osm-approvers  @openservicemesh/osm-docs-approvers any additional feedback?

**Review by nojnhuh — APPROVED** [review]
LGTM, just needs DCO signoff.

**Pull request merged**
